src/sr_od/app/sim_uni/sim_uni_app.py

```
@@ -76,28 +101,121 @@ def _check_times(self) -> OperationRoundResult:
+            op.execute()
+            # return self.round_by_op_result(op.execute())
+            state = sim_uni_screen_state.ScreenState.SIM_TYPE_NORMAL.value
+            return self.round_success(state)
+
+    @node_from(from_name='识别初始画面', status=sim_uni_screen_state.ScreenState.SIM_TYPE_X.value)  # 最开始已经在模拟宇宙入口了
+    @node_from(from_name='传送', status=sim_uni_screen_state.ScreenState.SIM_TYPE_X.value)  # 传送到差分宇宙, 调用差分宇宙自动化脚本
+    @operation_node(name='调用差分宇宙自动化')
+    def _execute_sim_universe_x(self) -> OperationRoundResult:
+        # 如果只要求打满奖励, 识别是否 14000/14000了
+        if self.ctx.sim_uni_config.only_points_reward:
+            for i in range(30):
```

Ruff (0.13.3)

Loop control variable `i` not used within loop body

(B007)